.github/ISSUE_TEMPLATE/question.yml

```
@@ -0,0 +1,36 @@
+name: Question
+description: Get help with errors or general questions
+title: "[QUESTION]"
+labels: ["question"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hey there, you have a question? We are happy to answer. Please make sure no similar question was opened already.
+
+        To make it easier for us to help you, please read this [article](https://git.io/JURJO) and try to follow the template below as closely as possible.
```

```suggestion
        To make it easier for us to help you, please read this [article](https://git.io/JURJO).
```

Because we can enforce the template now :D